plugins/mongodb.yaml

```diff
@@ -22,12 +62,34 @@ parameters:
     default: end
 
 # Set Defaults
+# {{$source := default "file" .source}}
+# {{$cluster_name := default "" .cluster_name}}
+# {{$pod_name := default "nginx-*" .pod_name}}
+# {{$container_name := default "*" .container_name}}
 # {{$log_path := default "/var/log/mongodb/mongodb.log*" .log_path}}
 # {{$mongodb_json_log_format := default false .mongodb_json_log_format}}
 # {{$start_at := default "end" .start_at}}
 
 # Pipeline Template
 pipeline:
+  # {{ if eq $source "kubernetes" }}
+  - id: kubernetes_input
+    type: kubernetes_container
+    pod_name: '{{ $pod_name }}'
+    container_name: '{{ $container_name }}'
+    cluster_name: '{{ $cluster_name }}'
+    start_at: '{{ $start_at }}'
+    enable_nested_json_parser: false
+    # {{ if $mongodb_json_log_format }}
+    write_to: log
```

Comment:
```suggestion
```
Looks like your  json_parser parses directly from record.

Reply:
Good catch. Initially I was going to add the write_to option to the container plugin, but I opted to remove $record.log from the mongo plugin instead: https://github.com/observIQ/stanza-plugins/pull/293